docs/commands/link.md

```diff
@@ -0,0 +1,18 @@
+### Description
+
+Gives you the link to the student's 42 intra with additional informations.
+
+- Full name
+- Campus
+- Level
+
+### Arguments
+
+- username:
+  - description: the student username
+  - required: ✅
+  - type: _string_
```

Comment:
Can you update the docs to match the command new changes. Thank you

Reply:
Sure!